src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -46,12 +46,13 @@ public function getLocale()
     /**
      * {@inheritdoc}
      */
-    public function getDomains()
+    public function getDomains(/* bool $excludeIntlDomains = true */)
```

Comment:
We should revert adding any arguments IMHO.
There is already a way to access intl+icu and regular domain message:

```php
$intlIcuMessages = $catalogue->all($domain.'+intl-icu');
$regularMessages = array_diff_key($catalogue->all($domain), $intlIcuMessages);
```

Reply:
Thanks @nicolas-grekas, that's a clever way to add ICU domain or not instead of modifying the MessageCatalogueInterface.

I've updated the PR!